src/commands/projects/add-pull-request.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * This command adds a pull request to a GitHub project column.
+ *
+ * @see: https://octokit.github.io/rest.js/#api-Projects-createCard
+ * const result = await octokit.projects.createCard({column_id, note, content_id, content_type})
+ * /projects/columns/:column_id/cards
+ */
+const flow = require('lodash.flow')
+
+const commonYargs = require('../../../lib/common-yargs')
+const printOutput = require('../../../lib/print-output')
+const authenticatedOctokit = require('../../../lib/octokit')
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([
+		commonYargs.withToken,
+		commonYargs.withJson,
+	])
+	return baseOptions(yargs)
+		.option('column_id', {
```

Comment:
We shouldn't be asking humans for internal IDs.

![image](https://user-images.githubusercontent.com/51677/55868975-709eb680-5b7d-11e9-9774-d1fad96fdb9a.png)

We should ask for the name/url of the project and the column to put it in. This is more inline with what we see in the UI.

We shouldn't expect users to know how to find GitHub's internal ID for a project board column.

Reply:
@see: https://github.com/Financial-Times/github/blob/52292e5cd86eea504742cef2af111fcbd2d4e83d/src/commands/projects/README.md